src/redux/features/plantings/plantingsSelectors.ts

```diff
@@ -58,25 +58,27 @@ export const selectPlantingProgress = createSelector(
     if (plantingSitesSearchResults && plantings) {
       const plantingsBySubzone = getTotalPlantsBySubzone(plantings);
       const totalPlantsBySite = getTotalPlantsBySite(plantingSitesSearchResults);
-      return plantingSites?.map((ps) => {
-        return {
-          siteId: ps.id,
-          siteName: ps.name,
-          reported: ps.plantingZones?.flatMap((zone) =>
-            zone.plantingSubzones
-              .filter((sz) => plantingsBySubzone[sz.id])
-              .map((sz) => ({
-                subzoneName: sz.fullName,
-                plantingCompleted: sz.plantingCompleted,
-                plantingSite: ps.name,
-                zoneName: zone.name,
-                targetPlantingDensity: zone.targetPlantingDensity,
-                totalSeedlingsSent: plantingsBySubzone[sz.id],
-              }))
-          ),
-          totalPlants: totalPlantsBySite[ps.id],
-        };
-      });
+      return plantingSites
+        ?.filter((ps) => totalPlantsBySite[ps.id])
```

Comment:
this is a one line change (added filter) rest of the change is code indent by formatter